RetailCoder.VBE/Inspections/Results/AssignedByValParameterInspectionResult.cs

```diff
@@ -25,9 +26,10 @@ public override IEnumerable<QuickFixBase> QuickFixes
         {
             get
             {
+                IAssignedByValParameterQuickFixDialogFactory factory = new AssignedByValParameterQuickFixDialogFactory();
```

Comment:
This latest update to the PR (2/25) is much closer (I'm hoping ) to what you expected to see.  The couple lines of code I added to RubberDuckModule.cs look out of place to me, so I suspect there is a better way.  As I'm sure you've already figured out, using Ninject is new to me...just another one of many things here at RD!.